be/src/exec/schema_scanner/schema_loads_scanner.cpp

```diff
@@ -0,0 +1,286 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "exec/schema_scanner/schema_loads_scanner.h"
+
+#include "exec/schema_scanner/schema_helper.h"
+#include "http/http_client.h"
+#include "runtime/runtime_state.h"
+#include "runtime/string_value.h"
+
+namespace starrocks {
+
+SchemaScanner::ColumnDesc SchemaLoadsScanner::_s_tbls_columns[] = {
+        //   name,       type,          size,     is_null
+        {"JOB_ID", TYPE_BIGINT, sizeof(int64_t), false},
+        {"LABEL", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"DATABASE_NAME", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"STATE", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"PROGRESS", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"TYPE", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"PRIORITY", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"ETL_INFO", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"TASK_INFO", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"CREATE_TIME", TYPE_DATETIME, sizeof(DateTimeValue), true},
+        {"ETL_START_TIME", TYPE_DATETIME, sizeof(DateTimeValue), true},
+        {"ETL_FINISH_TIME", TYPE_DATETIME, sizeof(DateTimeValue), true},
+        {"LOAD_START_TIME", TYPE_DATETIME, sizeof(DateTimeValue), true},
+        {"LOAD_FINISH_TIME", TYPE_DATETIME, sizeof(DateTimeValue), true},
+        {"JOB_DETAILS", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"ERROR_MSG", TYPE_VARCHAR, sizeof(StringValue), true},
+        {"TRACKING_URL", TYPE_VARCHAR, sizeof(StringValue), true},
+        {"TRACKING_SQL", TYPE_VARCHAR, sizeof(StringValue), true},
+        {"TRACKING_MSG", TYPE_VARCHAR, sizeof(StringValue), true}};
```

Comment:
ERROR_MSG is enough. ERROR_URL is not necessary because the IP is not accessible much times.

Reply:
In consideration of compatibility and convenient debugging, tracking url is still provided